### ISSUE TITLE
Value error during zipping of training batch

### ISSUE THREAD

**Issue by ahmeshaf** [dennybritz/cnn-text-classification-tf#47]
Hi,

I am trying to run train.py with my own dataset for a multi-class classification problem. I get a Value error during the generation of test batch. What can be the issue? It works fine on the provided +ve/-ve dataset
![screenshot from 2016-11-17 12-00-32](https://cloud.githubusercontent.com/assets/10776106/20378679/96df3a66-acbd-11e6-9090-53b26cb5763e.png)

**Comment by rehanwow9** [comment]
Ok. I found the issue.
In
`def batch_iter(data, batch_size, num_epochs, shuffle=True):` 
sometimes randomly start_index and end_index were becoming the same.
 so the overall 
`yield shuffled_data[start_index:end_index]`
 was returning an empty list
